demos/mqtt/mqtt_demo_mutual_auth/demo_config.h

```diff
@@ -112,6 +112,37 @@
  * #define CLIENT_PRIVATE_KEY_PATH    "...insert here..."
  */
 
+/**
+ * @brief The username string, required to authenticate to your MQTT broker.
+ *
+ * Refer to the AWS IoT documentation below for details regarding client
+ * authentication with username and password.
+ * https://docs.aws.amazon.com/iot/latest/developerguide/enhanced-custom-authentication.html
+ * An authorizer setup needs to be done as mentioned in the above link to use
+ * username/password based client authentication.
+ *
+ * @note AWS IoT message broker would require either a set of client certificate/private key
+ * or username/password to authenticate the client. If this config is defined, username and
+ * password will be used instead of client certificate and private key for client authentication.
+ *
+ * #define CLIENT_USERNAME    "...insert here..."
+ */
+
+/**
+ * @brief The password string, required to authenticate to your MQTT broker.
+ *
+ * Refer to the AWS IoT documentation below for details regarding client
+ * authentication with username and password.
+ * https://docs.aws.amazon.com/iot/latest/developerguide/enhanced-custom-authentication.html
+ * An authorizer setup needs to be done as mentioned in the above link to use
+ * username/password based client authentication.
+ *
+ * @note AWS IoT message broker would require either a set of client certificate/private key
+ * or username/password to authenticate the client.
```

Comment:
Should this statement be added here as well under `@note`?
"_If this config is defined, username and password will be used instead of client certificate and private key for client authentication._"

Reply:
The switch is based on CLIENT_USERNAME itself. That note is only required for CLIENT_USERNAME